src/content/troubleshooting/_index.en.md

```diff
@@ -58,7 +58,7 @@ If you are able to connect to remote service by using ServiceIP or globalIp, but
 This is good time to familiarize yourself with [Service Discovery Architecture](../architecture/service-discovery/) if you haven't already.
 
 ##### Check ServiceExport for your Service
-For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export.
+For a Service to be accessible across clusters, you must first export the Service via `subctl` which create a `ServiceExport` resource. Ensure the `ServiceExport` resource exists and check if its status condition indicates `Exported'. Otherwise, its status condition will indicate the reason it wasn't exported.
```

Comment:
```suggestion
For a Service to be accessible across clusters, you must first export the Service via `subctl` which creates a `ServiceExport` resource. Ensure the `ServiceExport` resource exists and check if its status condition indicates `Exported'. Otherwise, its status condition will indicate the reason it wasn't exported.
```